src/modules/universal-discovery/services/universal.discovery.service.js

```diff
@@ -193,7 +193,7 @@ export const findLocationsByParentLocationId = (
  *
  * @function findContentBySearchQuery
  * @param {Object} restInfo REST config hash containing: token and siteaccess properties
- * @param {String} query
+ * @param {Object} query Object containing query criterions
```

Comment:
Also, the `identifier` in `ViewInput` should be changed:
`udw-locations-by-search-query-${query}` => `udw-locations-by-search-query-${query.FullTextCriterion}